sky/clouds/cloud.py

```diff
@@ -66,6 +68,14 @@ class Cloud:
 
     _REPR = '<Cloud>'
 
+    _MAX_CLUSTER_NAME_LEN_LIMIT: Optional[int] = None
+
+    # All features implemented by this cloud. Used by
+    # Cloud.check_features_are_supported()
+    # None means all features are supported.
+    _CLOUD_IMPLEMENTATION_FEATURES: Optional[
+        Set[CloudImplementationFeatures]] = None
```

Comment:
Should None default to mean none supported? (Or, we can change default val to set().) 

Because we may add new values (for new features) to CloudImplementationFeatures in the future and that doesn't mean existing Clouds that default to None automatically support those.